app/controllers/articles_controller.rb

```diff
@@ -26,6 +26,8 @@ def create
   end
 
   def update
+    @article = Article.find(params[:id])
```

Comment:
create a `before_action` filter with name of `find_article` and put that line in that method